contracts/margined_engine/src/reply.rs

```diff
@@ -122,9 +123,22 @@ pub fn update_position_reply(
 
     let mut msgs: Vec<SubMsg> = vec![];
     let mut funds = read_sent_funds(deps.storage)?;
+    let fees = position.spread_fee.checked_add(position.toll_fee)?;
 
     // create transfer messages depending on PnL
-    if swap.margin_to_vault > Integer::zero() {
+    #[allow(clippy::comparison_chain)]
+    if swap.margin_to_vault < Integer::zero() {
```

Comment:
Using `swap.margin_to_vault.negative`

Reply:
resolve in https://github.com/oraichain/perpetuals/pull/2/commits/fd69e7dafc3ca81686407db27c212d80a0ac6586